tests/Test/DocumentsSpec.hs

```diff
@@ -36,6 +36,7 @@ spec =
         insertedConflict <- tryEsError $ insertData' cfg
         case insertedConflict of
           Right (res', _) -> liftIO $ isVersionConflict res' `shouldBe` True
+          Right _ -> liftIO $ expectationFailure "This should never not happen."
```

Comment:
Isn't it redundant?

Reply:
That's a *too much Git magic to create small PRs*-mistake. Sorry for the noise - I'm fixing it.